oap-server/oal-rt/src/main/java/org/apache/skywalking/oal/rt/parser/ConditionExpression.java

```diff
@@ -18,18 +18,42 @@
 
 package org.apache.skywalking.oal.rt.parser;
 
-import lombok.AllArgsConstructor;
+import java.util.LinkedList;
+import java.util.List;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 @Getter
 @Setter
 @NoArgsConstructor
-@AllArgsConstructor
 public class ConditionExpression {
     // original from script
     private String expressionType;
     private String attribute;
     private String value;
+
```

Comment:
I think this empty line is not required.

Reply:
done